client/src/components/Menu/index.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import Hamburger from 'react-hamburger-menu';
+import MenuItem from './MenuItem';
+
+import s from './Menu.css';
+
+class Menu extends React.Component {
+  static propTypes = {
+    currentTab: PropTypes.string.isRequired,
+  };
+
+  // eslint-disable-next-line react/state-in-constructor
```

Comment:
Is this comment needed? A.f.a.i.r. the rule was changed

Reply:
yes, im gonna remove it